### DISCUSSION TITLE
Migrate baseline storage to git-lfs

### DISCUSSION THREAD

**Discussion by mattpap** [bokeh/bokeh#11190] [General]
Discussion started in PR #11189. Currently baselines consist of 3000+ files and take under 27 MB of space. However, frequent updates of large subsets of those baseline files will eventually result in significant increase in size of the repository. If not LFS, then other suggestions are welcome.

**Comment by bryevdv** [comment]
I suppose another option is a sub-repo, etc. but *at a glance* git-lfs seems to operate a but more transparently. My own interests in any solution are 

* We may want to store some larger images at some point (speculating)
* We may want to store other large artifacts (e.g. the JSON output of `spec.py`) at some point
* It would be nice to be able to access baselines easily by chrome-version to run comparisons if necessary.
